src/app/core/pusher/pusher.service.ts

```diff
@@ -0,0 +1,179 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {select, Store} from '@ngrx/store';
+import {AppState} from '../store/app.state';
+import {Injectable, OnDestroy} from '@angular/core';
+import {environment} from '../../../environments/environment';
+import Pusher from 'pusher-js';
+import {selectCurrentUser} from '../store/users/users.state';
+import {UserModel} from '../store/users/user.model';
+import {Subscription} from 'rxjs/index';
+import {filter, take} from 'rxjs/internal/operators';
+import {AuthService} from '../../auth/auth.service';
+import {OrganizationsAction} from '../store/organizations/organizations.action';
+import {OrganizationConverter} from '../store/organizations/organization.converter';
+import {DocumentsAction} from '../store/documents/documents.action';
+import {convertDocumentDtoToModel} from '../store/documents/document.converter';
+import {ProjectsAction} from '../store/projects/projects.action';
+import {ProjectConverter} from '../store/projects/project.converter';
+import {ViewsAction} from '../store/views/views.action';
+import {ViewConverter} from '../store/views/view.converter';
+import {CollectionsAction} from '../store/collections/collections.action';
+import {CollectionConverter} from '../store/collections/collection.converter';
+import {ContactsAction} from '../store/organizations/contact/contacts.action';
+import {ContactConverter} from '../store/organizations/contact/contact.converter';
+import {ServiceLimitsAction} from '../store/organizations/service-limits/service-limits.action';
+import {ServiceLimitsConverter} from '../store/organizations/service-limits/service-limits.converter';
+import {PaymentsAction} from '../store/organizations/payment/payments.action';
+import {PaymentConverter} from '../store/organizations/payment/payment.converter';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PusherService implements OnDestroy {
+  private pusher: any;
```

Comment:
As @kubedo8 suggested, better approach would be to add [@types/pusher-js](https://www.npmjs.com/package/@types/pusher-js) as dev dependency to avoid using `any`.

Reply:
I tried that but it did not help somehow. It seemed to me like the types were for another library version/deployment